components/kyma-environment-broker/cmd/broker/broker_suite_test.go

```diff
@@ -122,6 +123,11 @@ func NewBrokerSuiteTest(t *testing.T, version ...string) *BrokerSuiteTest {
 	componentsYAML := kebRuntime.ReadYAMLFromFile(t, "kyma-components.yaml")
 	fakeHTTPClient := kebRuntime.NewTestClient(t, installerYAML, componentsYAML, http.StatusOK)
 
+	configProvider := kebConfig.NewConfigProvider(
```

Comment:
I'd like to see one constructor which initializes inside the reader and validator, so on the high level, you will have kebConfig.NewConfigProvider(ctx, cli, logrus.New())

Reply:
The idea was to explicitly show where the configuration comes from and how it's provided. On first sight we can see that ConfigProvider provides the configuration from a ConfigMap. I left some space for future implementations, for example, configuration provided from a file or some other endpoint. I think we can hide the components initialised in NewConfigProvider() if ConfigMap stays as a default source of the configuration.

Reply:
We discussed the matter and decided to keep the proposed solution